EDU-17893: Update CMS integration instructions (#2570)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/glossary.mdx

## V

### (VTEX) CMS

The [CMS](https://developers.vtex.com/docs/guides/cms-for-faststore-storefronts) is a VTEX content management system for defining, storing, and delivering storefront content.
The system follows a decoupled architecture that separates content authoring from content consumption. This separation allows content management workflows and content delivery workloads to evolve and scale independently.

### (VTEX) Headless CMS (legacy)

[Headless CMS (legacy)](https://developers.vtex.com/docs/guides/faststore/headless-cms-overview) is the legacy content management system for FastStore storefronts. It is a no-code system for creating, editing, and publishing web pages that delivers content through an intermediary data layer decoupled from the frontend.

> > ⚠️ Headless CMS is a legacy solution. New FastStore storefronts must use the [VTEX CMS](https://developers.vtex.com/docs/guides/cms-for-faststore-storefronts) instead. If you are setting up a new project, initiate it with the CMS from the start.

Reviewer: ```suggestion
> ⚠️ Headless CMS is a legacy solution. New FastStore storefronts must use the [VTEX CMS](https://developers.vtex.com/docs/guides/cms-for-faststore-storefronts) instead. If you're setting up a new project, initiate it with the CMS from the start.
```
